scripts/context-deploy-benchmark/context-benchmark.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+CONTEXT_AMOUNT=1
+
+run_contexts() {
+  for ((i = 1; i <= CONTEXT_AMOUNT; i++))
+  do
+    agc context deploy -c "$1$i";
```

Comment:
Capturing single context deployments is important, but we should also try to capture AGC's multi-context performance in a separate benchmark (e.g. `agc context deploy --all` or `agc context deploy -c "context1" -c "context2" ...`).

Today this almost equivalent to running them sequentially, but we have future enhancements coming that will be able to deploy them in parallel.